Makefile

```diff
@@ -7,26 +7,13 @@ CRD_DOCS_GENERATOR_VERSION=0.1.1
 
 default: docker-build
 
-build-css:
-	docker run -it \
-		-v ${PWD}/src/static/css:/sass \
-		ellerbrock/alpine-sass /sass/base.sass /sass/base.css -m auto
-
```

Comment:
Nice to get rid of sass as an extra dependency! Is this handled by HUGO directly?

Reply:
Yes. The "extended" edition of hugo includes sass compilation. :tada: